### PR TITLE
Add Chinese rate limits

### PR DESCRIPTION
**Pull request by keyganker**

### CODE REVIEW

File: locales/en.yml [hunk 1]
```diff
@@ -316,12 +314,32 @@ en:
...
+      </ul>
+    </li>
+    </ul>
+  ### Account rate limit
   rate_heading_understandin: Understanding Your Request Rate Limit
   rate_para_understanding: |
-    Every request to the API returns the fields shown in the code panel:
-  rate_text_limitStatus_understanding: your remaining requests
-  rate_text_limit_understanding: your current limit
+    Bybit makes frequency limits based on <b> the sliding time window per minute and uid</b>. Every request to the API returns the fields shown in the code panel:
```
Comment (keyganker) [inline locales/en.yml:340]: Add api limit description detail

File: locales/en.yml [hunk 2]
```diff
@@ -316,12 +314,32 @@ en:
...
   # API Rate Limits
   rate_heading: API Rate Limits
-  ### Understanding Your Request Rate Limit
+  ### IP Rate Limit
+  ip_rate_heading_understandin: IP Rate Limit
+  ip_rate_para_understanding: |
+    <p> Bybit has different IP frequency limits for <code>GET</code> and <code>POST</code> method：</p>
+    <ul>
+    <li>
+      <code>GET</code>method:
```
Comment (keyganker) [inline locales/en.yml:323]: Add IP limit description

File: source/includes/inverse_future/_rate_limits.md [hunk 3]
```diff
@@ -1,10 +1,13 @@
...
+### t(:ip_rate_heading_understandin)
+t(:ip_rate_para_understanding)
+
 ### t(:rate_heading_understandin)
 t(:rate_para_understanding)
 
 ```
 "rate_limit_status": 119,
-"rate_limit_reset_ms": 1572114055663815,
+"rate_limit_reset_ms": 1572114055663,
```
Comment (keyganker) [inline source/includes/inverse_future/_rate_limits.md:10]: There is a bug, it should be millisecond, not microseconds.
We'll fixed it before the new doc release, and I fix the doc here firstly

**Comment by keyganker** [comment]
@dexter-2 please have a review

File: source/includes/inverse_future/_market_data.md [hunk 4]
```diff
@@ -1,37 +1,6 @@
...
-
-```console
-curl https://api-testnet.bybit.com/v2/public/tickers
-```
-
-> t(:codequote_responseExample)
-
-```javascript
-{
-    "ret_code": 0,
```
Comment (keyganker) [inline source/includes/inverse_future/_market_data.md:15]: Move server time endpoint to API data category

**Comment by dextertd** [comment]
#6 

How should we use default.yml and en.yml? Should they be identical?

**Comment by keyganker** [comment]
> #6
> 
> How should we use default.yml and en.yml? Should they be identical?

default.yml is a copy of en.yml, it usd for default setting, for example:
default: http://host/inverse
en:http://host/en/inverse
zh-cn://host/zh-cn/inverse

And without default.yml, the path will be:
en:http://host/inverse
zh-cn://host/zh-cn/inverse

I don't know how to make the default path as "http://host/en/inverse"

**Comment by dextertd** [comment]
> default.yml is a copy of en.yml, it usd for default setting

I think this fixes it? http://host/inverse uses en.yml. #7

**Comment by keyganker** [comment]
> > default.yml is a copy of en.yml, it usd for default setting
> 
> I think this fixes it? http://host/inverse uses en.yml. #7

Cool! Thanks!

**Pull request merged**